__tests__/app.test.js

```diff
@@ -39,3 +39,46 @@ describe('GET/api/topics', () => {
       });
   });
 });
+
+describe('GET/api/articles', () => {
+  test('status 200, returns an array of article objects in correct format', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then((result) => {
+        const articles = result.body.articles;
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
```

Comment:
another great assertion here is to check that the 'body' property is not present in each article